### PR TITLE
Updated building docs regarding CMake update

### LINKED ISSUE #2577: Sounds from chatterino won't play without `pulseaudio-qt` package on Arch Linux
Opening this issue since I've got myself a very barebones, fresh Arch installation and tried building from source while following docs in the repo. While I've configured audio properly for all other apps, chatterino couldn't play my highlight sounds due to `pulseaudio-qt` package not being installed.
I am asking whether we should we add this to building docs or make a wiki entry about it since not everyone builds from source, but they might be missing the package.

### PR DESCRIPTION
**Pull request by zneix**
Pull request checklist:

- [x] `CHANGELOG.md` was updated, if applicable
no changelog entry needed

# Description

- Updated required packages on Arch GNU/Linux It's me Leon, not mm pepeLaugh
- Updated required packages on Ubuntu

I think ti's safe to assume this closes #2577

### CODE REVIEW

File: BUILDING_ON_LINUX.md [hunk 1]
```diff
@@ -6,8 +6,25 @@ Note on Qt version compatibility: If you are installing Qt from a package manage
...
+
+1. Install C++ IDE Qt Creator `sudo apt install qtcreator`
+1. Open `chatterino.pro` with Qt Creator and select build
+
+### Manually
+
+1. go into project directory
+1. create build folder `mkdir build && cd build`
+
+### Using QMake
```
Comment (pajlada) [inline BUILDING_ON_LINUX.md:21]: Let's indent it further :)
```suggestion
#### Using QMake
```

File: BUILDING_ON_LINUX.md [hunk 2]
```diff
@@ -6,8 +6,25 @@ Note on Qt version compatibility: If you are installing Qt from a package manage
...
+
+1. `qmake .. && make`
+
+### Using CMake
```
Comment (pajlada) [inline BUILDING_ON_LINUX.md:25]: Let's indent it further :)
```suggestion
#### Using CMake
```

**Review by pajlada — CHANGES_REQUESTED** [review]
two smol changes, please commit those in if they seem good and we can get this merged in :)

**Comment by zneix** [comment]
I Resolved the indentation issues :+1: Also added a note about `pulseaudio-qt` package being an optional dependency for Arch GNU/Linux - it's required (on pulseaudio systems) for chatterino to output audio on events like highlights and live notifications

File: BUILDING_ON_LINUX.md [hunk 3]
```diff
@@ -17,18 +34,22 @@ _most likely works the same for other Debian-like distros_
...
 
-### Using CMake
+#### Using CMake
 
 1. `cmake .. && make`
 
+### Optional dependencies
+
+1. `sudo pacman -S pulseaudio-qt` _(optional: for audio output)_
+
```
Comment (pajlada) [inline BUILDING_ON_LINUX.md:52]: Let's leave this for a future PR since it requires a bit more testing

```suggestion
```

**Review by pajlada — APPROVED** [review]
LGTM :+1:

**Pull request merged**